File: Karan-Example-Project-Solutions/Python/Text/PigLatin.py
import string

def convert(string):
    if string[0] in "aeiou":
        return string + 'yay'
    else:
        return string[1:] + string[0] + 'ay'

user = str(input("Enter sentence to convert: "))
user = user.translate(str.maketrans('', '', string.punctuation))
user = user.translate(str.maketrans('', '', '1234567890'))
ls= user.split(" ")

print(' '.join(convert(elem) for elem in ls))

File: Karan-Example-Project-Solutions/Java/Text/PigLatin.java
import java.util.Scanner;
import java.util.Arrays;

public class PigLatin {

  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);
    System.out.println("Enter word to convert: ");
    String in = input.nextLine();

    in = in.replaceAll("[^a-z\\sA-Z]","");

    System.out.println(convert(in));
  }

  private static String convert(String in) {
    String[] array = in.split(" ");

    for (int i = 0; i < array.length; i++) {
      if ("AEIOUaeiou".indexOf(array[i].charAt(0)) < 0) {
        array[i] = array[i].substring(1) + array[i].charAt(0) + "ay";
      } else {
        array[i] += "yay";
      }
    }

    return Arrays.toString(array);
  }
}
